policies/tiles.md

```diff
@@ -5,75 +5,188 @@ permalink: /policies/tiles/
+**You must not:**
 
-Use of any OSMF provided service is further governed by the [OSMF Terms of Use](https://wiki.osmfoundation.org/wiki/Terms_of_Use).
+* Bulk download ("scrape") tiles or offer prefetch features.
+* Send `Cache-Control: no-cache`, `Pragma: no-cache`, or similar no-cache headers by default.
+* Set a restrictive Referrer-Policy that prevents the **HTTP Referer** header being sent.
+* Masquerade as another app's User-Agent, or rely on a library's default User-Agent.
 
-## Usage
+**You should (recommended):**
+
+* Avoid hard-coding the tile URL; allow switching without needing a software update.
+* Add a "Report a map issue" link to [https://www.openstreetmap.org/fixthemap](https://www.openstreetmap.org/fixthemap).
+* Publish a contact email on your website or app store listing.
+* Support HTTP/2 or HTTP/3 for efficient multiplexed downloads.
+
+---
 
-The correct URL for the tiles is `https://tile.openstreetmap.org/{zoom}/{x}/{y}.png`. Other subdomains are not recommended as they may be slower or discontinued in the future. We are unable to offer support on how to make your preferred map library work with this URL and recommend you consult its documentation.
+## 1. Correct tile URL
 
-## Requirements
+Use exactly:
 
-* Clearly display [license](https://wiki.openstreetmap.org/wiki/License) [attribution](https://wiki.osmfoundation.org/wiki/Licence/Attribution_Guidelines), normally in the bottom-right corner of the map.
+```
+https://tile.openstreetmap.org/{z}/{x}/{y}.png
+```
+
+Other subdomains or hostnames may be slower or withdrawn without notice. We cannot provide integration support for specific map libraries; please consult your library's documentation.
+
+---
+
+## 2. Attribution and licensing
+
+* Show OpenStreetMap licence attribution clearly on the map (typically bottom-right).<br>
+  Typically: *© [OpenStreetMap](https://www.openstreetmap.org/copyright) contributors*<br>
+  See: [Attribution Guidelines](https://wiki.osmfoundation.org/wiki/Licence/Attribution_Guidelines)
 * Do not actively or passively encourage copyright infringement.
-* Recommended: Do not hardcode any URL to tile.openstreetmap.org as doing so will limit your ability to react if the service is disrupted or blocked. In particular, switching should be possible without requiring a software update.
-* Recommended: add a link to [https://www.openstreetmap.org/fixthemap](https://www.openstreetmap.org/fixthemap) to allow your users to report and fix problems in our data.
-* Recommended: a contact email on your web page or app store page. If we cannot find contact information we will be unable to do anything except block if an issue arises.
+* Do not hide attribution beneath UI, behind toggles, or off-screen.
+
+---
+
+## 3. Technical requirements
+
+### 3.1 Identification
+
+* **User-Agent:** Send a clear, unique User-Agent string that names your app and includes a contact URL or email.<br>
```

Comment:
If they've got contact information on the site or app page I don't think we've ever had problems finding them based on a user-agent. I think the current UA requirements are fine.

We have the "may try to contact you" section that covers being able to contact them based on the info they give.